pkg/kustomize/secrets_test.go

```diff
@@ -285,7 +285,7 @@ var containsComponentConfigTests = []struct {
 		"route",
 		true,
 		`SERVER_HOSTNAME: registry.skynet.com`,
-		false,
+		true,
```

Comment:
The same question is also valid here, why the behavior is different now?

Reply:
This is a refactor of this function where it will only return whether or not a field for a given component is present. The logic of allowing Route fields such as `SERVER_HOSTNAME` to be set now happens outside of this function.